tests/main.test.ts

```diff
@@ -113,7 +113,7 @@ describe("User start/stop", () => {
   });
 
   test("Author's manual unassign should close linked issue", async () => {
-    const infoSpy = jest.spyOn(console, "info").mockImplementation(() => {});
+    const infoSpy = jest.spyOn(console, "info").mockImplementation(() => { });
```

Comment:
@Keyrxng did you run prettier on this file and this happened?

Reply:
my bad must have forgot to run it after saving